src/Libs/GLib-2.0/Public/ExceptionManager.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+
+namespace GLib;
+
+public static class ExceptionManager
```

Comment:
I would prefer a little more descriptive class name like `UnhandledException`. The event could be named `Raised` and the method to raise could be named `Raise`.

Code would look like:

```CS
try
{
    _callback(returnValue, paramValues);
}
catch (Exception e)
{
    GLib.UnhandledException.Raise(e);
}
```

```CS
GLib.UnhandledException.Raised += (_, args) =>
{
    caught_exception = args.ExceptionObject is NotImplementedException;
};
```

As the class is part of the public API we should be generously document the code for a potential user of the library.

Possible summary for the "UnhandledException" class:
> Allows to handle exceptions which can't cross the native code boundary and would crash the runtime.

Event summary:
> Is invoked if an exception is raised which can't be transferred into managed code. This allows to handle those exceptions gracefully and avoid a runtime crash.

Method summary:
> Call this method to raise the "Raised" event. Exists the runtime if the event has no subscribers.

Method remark:
> This method is not intended for users to call directly but only for code which catches an exception which would otherwise crash the runtime as the caught exception can not cross the native code boundary.